vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -64,6 +64,9 @@
   <ImportGroup Label="ExtensionSettings">
   </ImportGroup>
   <ImportGroup Label="Shared" />
+  <Import Project="..\Chakra\Chakra.vcxitems" Label="Shared" />
+  <Import Project="..\JSI\Shared\JSI.Shared.vcxitems" Label="Shared" />
```

Comment:
>JSI [](start = 22, length = 3)

(nit) Just curious if we should do casing as suggested 
[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)

eg: Jsi instead of JSI
eg: Uwp intead of UWP

Reply:
We are just at the beginning of this project, so it would be best time to set on naming conventions :)

---
In reply to: [360530124](https://github.com/microsoft/react-native-windows/pull/3809#discussion_r360530124) [](ancestors = 360530124)

Reply:
I agree with you. As we progress with the code we must fix all of this.

---
In reply to: [360530677](https://github.com/microsoft/react-native-windows/pull/3809#discussion_r360530677) [](ancestors = 360530677,360530124)